### PR TITLE
feat: upgrade terraform version to 0.13.4 and terragrunt to 0.25.3

### PR DESCRIPTION
**Pull request by julienpierini**

### CODE REVIEW

File: config.yml [hunk 1]
```diff
@@ -439,6 +439,21 @@ terraform:
         - git --version
     template_vars:
       BASE_IMAGE_VERSION: 3.8
+  "0.13.4":
```
Comment (clemstoquart) [inline config.yml:442]: Hello, I think you can override the version `0.13.1` instead of creating a new one.

**Review by clemstoquart — CHANGES_REQUESTED** [review]

File: config.yml (hunk 1 above)
Comment (Mallear) [reply config.yml:442]: Terraform new versions aren't retro-compatible. We need to maintain multiple minor versions.

Comment (Mallear) [reply config.yml:442]: I think we can stop maitaining 0.12.29. Can you confirm @axelpavageau ?

Comment (clemstoquart) [reply config.yml:442]: Ok 😢

File: config.yml [hunk 2]
```diff
@@ -439,6 +439,21 @@ terraform:
...
+    build_args:
+      TERRAFORM_VERSION: 0.13.4
+      TERRAGRUNT_VERSION: 0.25.3
```
Comment (Mallear) [inline config.yml:445]: You can upgrade other image's terragrunt version.

**Review by clemstoquart — APPROVED** [review]

File: config.yml [hunk 3]
```diff
@@ -439,6 +439,21 @@ terraform:
...
+      TERRAFORM_COMPLIANCE_VERSION: 1.3.2
+    test_config:
+      cmd:
+        - terraform version
+        - terragrunt --version
+        - terraform-compliance --version
+        - python --version
+        - aws --version
+        - git --version
```
Comment (Mallear) [inline config.yml:454]: Can you use YAML anchors to declare this block of cmd only once ?

**Pull request merged**